dev/src/Helpers.ts

```diff
@@ -276,6 +279,32 @@ export const generateAssignmentsHeader = (base: RowElement[], assignments: Assig
     return base
 }
 
+const slugify = (str: string): string => {
```

Comment:
We shouldn't need to call this in the frontend. That is, we should instead make sure that students can only type allowed characters in the group name, and thus their github group repo name can be used directly on Line 306.